static-site/content/blog/2024-08-30-augur-v25-features.md

```diff
@@ -0,0 +1,122 @@
+---
+author: "The Nextstrain team"
+date: "2024-08-30"
+title: "Notable changes in Augur v25"
+---
+
+The Nextstrain team has released several new versions of Augur – our
+bioinformatics toolkit that assists with phylogenomic analyses – over
+the past few months. We wanted to highlight some of the significant
+feature improvements in these new releases.
+
+### Excel and OpenOffice support
+
+One of the features we are most excited to announce is that the
+[`augur curate`][] command now supports both Excel (`.xlsx` and
+`.xls`) and OpenOffice (`.ods`) files as metadata inputs. This allows
+the easy conversion of Excel or OpenOffice files into the expected
+metadata TSV used by other Augur commands. *This new feature is
+available in Augur v25.2 and later.*
+
+### Merging multiple metadata files
+
+We are also very pleased to announce a new command, [`augur merge`][],
+which allows for generalized merging of two or more metadata tables
+based on a common field. We hope that this will make curation of
+metadata inputs significantly easier. We also expect that this will
+facilitate starting with existing pathogen repos with curated data
+available via [data.nextstrain.org][] and “spiking in” extra,
+non-public metadata for additional analysis. We plan to extend this
+functionality to support merging of sequence files in a future Augur
+release. *The merge feature is available in Augur v25.3 and later.*
+
+### Weighted filtering
+
+We have extended `augur filter` with a new flag,
+[`--group-by-weights`][], that adds weighted subsampling. This allows
```

Comment:
Sorry I didn't catch this when it was added in the initial draft. I've called this "weighted sampling" in the [guide](https://docs.nextstrain.org/en/latest/guides/bioinformatics/filtering-and-subsampling.html) and it would be good to keep consistent.

```suggestion
### Weighted sampling

We have extended `augur filter` with a new flag,
[`--group-by-weights`][], that adds weighted sampling. This allows
```